README.md

```diff
@@ -108,6 +108,18 @@ such a server can be launched with `python -m smtpd -n -c DebuggingServer localh
 
 Alternatively only unit tests can be run by doing `cargo test --lib`.
 
+## Troubleshooting
+
+These are general steps to be followed when troubleshooting SMTP related issues.
+
+- Ensure basic connectivity, ensure requisite ports are open and daemons are listening.
+- Confirm that your service provider allows traffic on the ports being used for mail transfer. (traffic on ports 25, 2525, 587, and 465 is commonly blocked by some service providers.)
```

Comment:
That doesn't sound right? Do you have any providers where this is the case beyond outbound on port 25?

- Outbound traffic on port 25, to send mail yes.
- Inbound traffic to receive mail on port 25 should be fine.
- 587 and 465 are _submission_ (starttls, explicit TLS) and _submissions_ (implicit TLS) ports (_submission/submissions are service aliases to the ports, usually defined in `/etc/services`_). These require auth to submit mail, unlike port 25. Port 465 historically was SMTPS (equivalent to port 25 but with implicit TLS), but that purpose was changed back in 2018 IIRC, changing it to a submissions port.

Reply:
Whoa those are really good docs.
Sorry, I'm thinking in terms of residential ISPs rather than commercial hosting services.

I think removing the entire parenthetical would be best.